health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CreateCampaign.js

```
@@ -0,0 +1,197 @@
+    config: {
+      enable: false,
+    },
+  };
+
+  const mutation = Digit.Hooks.useCustomAPIMutationHook(reqCreate);
+
+  const  handleValidate = (formData) => {
+    const key = Object.keys(formData)?.[0];
+    switch (key) {
+      case "DateSelection":
+        const startDateObj = new Date(formData?.DateSelection?.startDate);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)